backend/nomad.go

```diff
@@ -137,3 +180,19 @@ func (n *Nomad) watchNodes() {
 		q = &api.QueryOptions{WaitIndex: waitIndex}
 	}
 }
+
+func (n *Nomad) watchMembers() {
+	for {
+		members, err := n.MembersWithID()
+		if err != nil {
+			log.Errorf("watch: unable to fetch members: %s", err)
+			time.Sleep(10 * time.Second)
+			continue
+		}
+
+		n.members = members
+		n.updateCh <- &Action{Type: fetchedMembers, Payload: members}
+
+		time.Sleep(1 * time.Second)
```

Comment:
It matters because I don't want to risk overloading the Nomad server in any way. I can't properly assess what kind of load it will generate and it will fill up the server logs (output) quickly. That's where the hesitation comes from. But I can merge it and will fix it later if that's ok with you.